controller/controller_test.go

```diff
@@ -1121,87 +1132,98 @@ func TestDeleteRecyclesIP(t *testing.T) {
 		t.Fatal("svc2 didn't get an IP")
 	}
 }
-func TestControllerDualStackConfig(t *testing.T) {
```

Comment:
don't we want to keep testing the dual-stack scenario?
I think that with the changes you introduce, we're removing the 
check that the controller assigns dual stack ip for a service.